flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/integration/test/FlinkEnvIT.java

```diff
@@ -212,4 +254,14 @@ protected static void clearTableInSchema() {
       TestUtils.assertTableResult(deleteResult, ResultKind.SUCCESS);
     }
   }
+
+  private String getIcebergRestServiceUri() {
+    JettyServerConfig jettyServerConfig =
+        JettyServerConfig.fromConfig(
+            serverConfig, String.format("gravitino.%s.", icebergRestServiceName));
+    return String.format(
+        "http://%s:%d/iceberg/", jettyServerConfig.getHost(), jettyServerConfig.getHttpPort());
+  }
+
+  protected abstract String getProvider();
```

Comment:
Could you move this method after `flinkByPass` method?

Reply:
ok